invokeai/app/api_app.py

```diff
@@ -12,12 +12,11 @@
 from fastapi_events.middleware import EventHandlerASGIMiddleware
 from pydantic.schema import schema
 
-from ..backend import Args
 from .api.dependencies import ApiDependencies
 from .api.routers import images, sessions, models
 from .api.sockets import SocketIO
-from .invocations import *
```

Comment:
It's important to have this line to import all invocations properly ahead of the API docs being generated. Do things work without this? I suspect if so, it's because they're getting imported through some other module, but it'd be a good idea to have them imported here as well for safety.

Reply:
I removed here and in other places because `pyflakes` claimed that the module symbols could not be imported and the line was being ignored. Everything seems to work without it, but can put back. How do I check?

Reply:
It does some fancy Python things to find and import all invocations when you have that line. I'm guessing pyflakes probably just doesn't follow the logic. There are a few places that have that line, but having it run before certain other things run is important.